pkg/validation/validation.go

```diff
@@ -0,0 +1,74 @@
+package validation
+
+import (
+	"fmt"
+)
+
+const (
+	ascii0 = 48
+	ascii9 = 57
+	asciiA = 65
+	asciiZ = 90
+	asciia = 97
+	asciiz = 122
+)
+
+// ValidateAccount checks that your account name is valid
+//
+// the identifier must start with an alphabetic character and cannot contain spaces or special characters unless
+// the entire identifier string is enclosed in double quotes (e.g. "My object").
```

Comment:
We (try to) always use the double-quote form, meaning that these validations are not relevant.

Reply:
I originally read this as 
```
the identifier must start with an alphabetic character && 
(! contain spaces or special characters || the entire identifier string is enclosed in double quotes)
```

But I just checked and "1test" is valid, so yes I'll go ahead and remove this.